content/en/docs/deployment/releases.md

```diff
@@ -77,6 +77,30 @@ You can build specific releases using `rebar3 release -n <release_name>`
 
 ## Build Configuration
 
+### App Syntax {id="app-syntax"}
```

Comment:
The [explicit heading ID](https://gohugo.io/content-management/cross-references/#heading-ids) here  is to indicate to future editors of this line that `app-syntax` is to be kept in-sync with the places which refer to it as `#app-syntax` 🙂